app/src/main/java/com/googleresearch/capturesync/SoftwareSyncController.java

```diff
@@ -302,20 +338,25 @@ private void setupSoftwareSync() {
         @RequiresApi(api = Build.VERSION_CODES.N)
         @Override
         protected Void doInBackground(Void... voids) {
+            Log.d(TAG, "Starting AlignPhasesTask");
```

Comment:
Lets extract this asynctask to separate file

Reply:
It works with `SoftwareSyncController`'s private fields such as `mState` and `mIsPeriodCalculated`, which, as I think, is not a good idea to expose.

I also noticed that `AsyncTask` became deprecated in API 30.

Reply:
Ok, I agree